packages/sdks/core-js-sdk/src/sdk/types.ts

```diff
@@ -276,6 +281,8 @@ export type FlowResponse = {
     options: string;
     create: boolean;
   };
+  // native data - if the action is 'native'
+  native?: string;
```

Comment:
I don't quite relates to the term `native` 

Can you convince me why you chose this name as the action name ?

Also, what this string holds? A JSON string? If so? Shouldn't we get it as a such?